scripts/deploy.sh

```diff
@@ -1,95 +0,0 @@
-#!/bin/bash
```

Comment:
This was using Travis to automatically deploy to train and production, correct? Does the new continuous integration do this for us?

Reply:
Yes, see the CI/CD section of the README for details.